corehq/apps/export/models/new.py

```diff
@@ -148,13 +148,15 @@ def create_default_from_export_item(group_schema_path, item, app_ids_and_version
         """
 
         is_main_table = group_schema_path == MAIN_TABLE
+        is_advanced = isinstance(item, SystemExportItem) and item.is_advanced
```

Comment:
yeah one thought would be to move that to the `ExportColumn`. and not make `item` a field on `ExportColumn` that way we don't have to have an item for each column. So instead of SystemItems they'd be SystemColumn. i haven't looked at all the ways we use `column.item` but it seems that the majority of it is just accessing the `path` which we can copy over. it'd definitely be a bit of a refactor and not for this PR but something we may want to do if we think it's worth it to keep the abstraction clean (while we still have a lot of context on this code)